src/components/textbox/textbox.component.tsx

```diff
@@ -114,8 +120,6 @@ export interface TextboxProps extends CommonTextboxProps {
   isOptional?: boolean;
   /** Container for DatePicker or SelectList components */
   positionedChildren?: React.ReactNode;
-  /** Label id passed from Select component */
-  labelId?: string;
```

Comment:
`labelId` was actually a publicly-documented prop, so removing it is a breaking change and should therefore be labelled as one.

I'm also not sure why we have to remove this - while it's inconsistent that Textbox has it and no other input components do, consumers might be using it to eg assemble their own `aria-labelledby` much as you are now suggesting for `InlineInputs`

Reply:
you're right, this prop should be documented as internal in the first place. I will add a breaking change disclaimer here.

As for the removal of it - customers could still compose their own aria-labelledby as long as they provide their own id for the component, as the label id is created using the `{id}-label` pattern, which is documented in the `id` property of this component.